lib/src/io_client.dart

```diff
@@ -3,37 +3,24 @@
 // BSD-style license that can be found in the LICENSE file.
 
 import 'dart:async';
+import 'dart:io';
```

Comment:
As of 1.22, `dart:io` is importable on all platforms and will just throw `UnsupportedError`s at runtime if it's not supported.